packages/appstore/package.json

```diff
@@ -25,7 +25,7 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@deriv/account": "^1.0.0",
-        "@deriv/api-types": "^1.0.54",
+        "@deriv/api-types": "^1.0.11",
```

Comment:
same here

Reply:
Because it is failing some of the test cases in codecov and it doesn't have that specific call unless you update it.